src/app/messages/en/messages.json

```diff
@@ -86,6 +86,12 @@
   "i18n:search-results:bar:query:results": "{total, plural, one {search result for } other {search results for }}",
   "i18n:search-results:bar:search-icon:value": "Search",
   "i18n:search-results:bar:toggle-text:mobile": "Search results",
+  "i18n:studyguides:popup:color-key": "Color key",
+  "i18n:studyguides:popup:color-key:terms:aria-label": "{color} key term",
```

Comment:
i don't think this message is right, is it in the spec? we might need malar or suhani to come up with something